tasks/push-snapshot/tests/mocks.sh

```diff
@@ -60,6 +60,12 @@ function date() {
       "+%s")
           echo "1696946200" | tee $(workspaces.data.path)/mock_date_epoch.txt
           ;;
+      "+%Y-%m-%d")
+          echo "1980-01-01" | tee $(workspaces.data.path)/mock_date_epoch.txt
```

Comment:
Why would you save this value to `$(workspaces.data.path)/mock_date_epoch.txt`? It's confusing, because it's not the epoch, but also, it doesn't seem like your tests use this file anyway. So maybe just remove the `| tee` part?

Same with the next one below.

Reply:
fixed